packages/prisma/schema.prisma

```diff
@@ -171,6 +171,8 @@ model Credential {
   selectedCalendars    SelectedCalendar[]
   invalid              Boolean?              @default(false)
   CalendarCache        CalendarCache[]
+  /// Note: settings type depends on implementation/slug this credential belongs to.
+  settings             Json                  @default("{ \"empty\": \"true\" }")
```

Comment:
The architecture of the audit logs feature will enable users to decide which events are reported through a nice UI. This is part of the default internal behaviour and disabled events are stored in Credential.settings. 

Settings is used to store an object of type: { disabledEvents: AuditLogTriggerEvents[] }, handleAuditLogTrigger will loop through all credentials and report the event if event is not found in disableEvents. 

Having settings stored directly in the app’s Credential allows for isolation of preferences between app installations.

For example, BoxyHQ Retraced could be installed several times per user/team/organization and each installation could have its own stream/project to report to, and unique preferences as to what to report and what not to report. 

I considered using App.settings as the storage, but this would define settings globally to all BoxyHQ Retraced instances, disabling isolation.